### PR TITLE
Add READMEs to NPM packages

### PR DESCRIPTION
**Pull request by axelboc**

### CODE REVIEW

File: packages/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "npm": "6.x"
   },
   "scripts": {
-    "prebuild": "rm -Rf dist-*",
+    "prebuild": "rm -Rf \"*/dist\"",
```
Comment (loichuder) [inline packages/package.json:17]: `rm -rF ./*/dist` does not work ?

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 [![Demo](https://img.shields.io/website?down_message=offline&label=demo&up_message=online&url=https%3A%2F%2Fh5web.panosc.eu%2F)](https://h5web.panosc.eu/)
 
-H5Web is web-based viewer to explore HDF5 files. It is built with React and uses
+H5Web is a web-based viewer to explore HDF5 files. It is built with React and uses
```
Comment (loichuder) [inline README.md:5]: :fishing_pole_and_fish:

File: .gitignore [hunk 3]
```diff
@@ -2,7 +2,7 @@ node_modules/
 
 /build/
 /storybook-static/
-/packages/dist-*/
+/packages/**/dist/
```
Comment (loichuder) [inline .gitignore:5]: `/packages/*/dist/` does not work as in `.eslintignore` and `.prettierignore` ?

**Review by loichuder — APPROVED** [review]

File: .gitignore (hunk 3 above)
Comment (axelboc) [reply .gitignore:5]: The question is more does `/*/` work in `.eslintignore` and `.prettierignore`... 🤣 🎣

File: packages/package.json (hunk 1 above)
Comment (axelboc) [reply packages/package.json:17]: Windows likes globs to have double quotes.

Comment (axelboc) [reply packages/package.json:17]: Actually, trying it again now, it doesn't work with the quotes... Removing them now.

**Pull request merged**